src/proto/engines.proto

```diff
@@ -26,6 +26,7 @@ message EngineInfo {
   EngineTokenizer tokenizer = 5;
   string name = 6;
   string description = 7;
+  bool can_fine_tune = 8;
```

Comment:
Does this nomenclature still make sense if the FT engine is separate? It's not `can` it's `is` in that case, right? As you wouldn't run normal inference on the fine tune-enabled engine.

Reply:
We actually don't have separate engines in the engines table for fine-tuning. Users can query which engines can be fine-tuned using this flag.